sycl/doc/extensions/proposed/sycl_ext_accessor_get_buffer.asciidoc

```diff
@@ -0,0 +1,93 @@
+= sycl_ext_accessor_get_buffer
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Introduction
+IMPORTANT: This specification is a draft.
+
+NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
+
+This document describes an extension that adds features for SYCL work items and groups to be available globally.
```

Comment:
This looks like a copy-paste error, copied over from another extension.  You should replace this with a brief summary of what this extension is for, or just remove the line (because everything is already in "Overview").